### ISSUE TITLE
[Suggestion/Feedback] Rename the "Wendigo"

### ISSUE THREAD

**Issue by lycannibal** [Alex-the-666/Ice-and-Fire-Suggestions#1199]
The subject of the wendigo can sometimes be a sensitive one, but as a Native American individual speaking on behalf of those of us who take issue with this myth's modern representation, I ask that renaming the creature currently being referred to as a "wendigo" be strongly considered.

In traditional Algonquian folklore, the wendigo is a mythological creature which is so reprehensible and hated that some are unwilling to even speak its name for fear it invites it. The wendigo represents what is often considered one of the most evil crimes a human being can commit, the act of cannibalism, and in particular the way starvation in the cold winter months can push some people to commit this vile act. This phenomenon, despite its rarity, and the legends of the wendigo surrounding it have lead to a long history of stereotypical portrayals of Native American and First Nations individuals as cannibals, used to reinforce the racist view of us as savages who do horrible things. White people also appropriated this myth into the term "wendigo psychosis," a racist pseudo-science so-called mental illness thought to be an insatiable craving for human flesh which exclusively Native American people experienced.

Additionally, it is worth noting that the current preview model for the "wendigo" is completely inaccurate to its depiction in traditional mythology, instead resembling the currently popular but completely baseless appearance they are often given in media. This form of cultural misappropriation can be harmful to those individuals who still hold this mythology as an important part of their lives.

TL;DR Please strongly consider choosing a different name for the creature currently being referred to as a wendigo rather than stripping a traditional Native American myth of its cultural context and misrepresenting it, for the sake of Native American members of the community like myself.

**Comment by LordDecay26** [comment]
i wouldnt get my hopes up, brother.
back when the ""wendigo"" was first announced i made the same suggestion, i was laughed into silence by the community and flat-out ignored by Alex and Raptorfarian.

**Comment by LordDecay26** [comment]
btw, i dont really think the design is harmful in any way but im just kinda tired of seeing this popular mix-up
i think what happened is the modern depiction of a wendigo is a chimera of both the skinwalker and the wendigo, as most websites online say they're the same thing, which we all know they are not.

**Comment by LAB5565** [comment]
Exactly. In traditional folklore, the skinwalker is actually a witch(I think, It's been a while since I read up on them) that takes the form of the animal hides they wear on them, while a Wendigo is, as mentioned above, an entirely separate entity from the skinwalker.
Although I admit I somewhat get why they would use the more popular design for the Wendigo. It's currently a more recognized version of the Wendigo, and they might be a bit uncomfortable using the traditional design for the mod.

**Comment by LordDecay26** [comment]
the wendigo is something of the Algonquin version of "Satan" whereas the skinwalker is a Witch who gains their powers of transformation through doing something evil, usually killing a family member for money or cannibalism, they are able to take the form of any animal they want, so basically a souped-up werewolf. sometimes though, when they transform back into a human, they aren't able to fully return resulting in them having animal features, this could be something as subtle as eyes or something as apparent as their entire head.

i think both the actual concept of THE wendigo and skinwalkers are far more interesting than the pop-culture versions of them,
especially because there are some parallels between those with crow/raven medicine and wendigos, they are both said to be able to shift their forms at will, and are masters of disguise and i've always wondered if there was some connection, like maybe only those with crow medicine could become a skinwalker.

**Comment by SlushieGamez** [comment]
WOW read this and my heart rate went up like 20 percent. lol
